### PR TITLE
Fix makefile

### PR DESCRIPTION
**Pull request by wirazeta**
# Changelog

- Create Task Domain
- Create Task Entity
- Trying To Create Task Use Case

## Related JIRA Card

- Create Task Function

## The changes contain

- [x] new features
- [ ] bugfix
- [ ] code improvements
- [ ] hotfix
- [ ] breaking changes

### CODE REVIEW

File: env/docker-compose.yml [hunk 1]
```diff
@@ -9,26 +9,26 @@ services:
...
+  #   restart: on-failure
+  #   ports:
+  #     - "3306:3306"
+  #   command: mysqld --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
+  #   volumes:
+  #     - "./storage/mysql/scripts:/docker-entrypoint-initdb.d"
+  #     - "./storage/mysql/data:/var/lib/mysql"
+  #   environment:
+  #     MYSQL_ROOT_PASSWORD: password
+  #     MYSQL_TCP_PORT: 3306
```
Comment (adiatma85) [inline env/docker-compose.yml:31]: Iki jarno di uncomment ae Wir

File: src/business/usecase/task/task.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+// GetList implements Interface.
+func (*task) GetList(ctx context.Context, params entity.TaskParam) ([]entity.Task, error) {
+	panic("unimplemented")
+}
+
+// Update implements Interface.
+func (*task) Update(ctx context.Context, updateParam entity.UpdateTaskParam, selectParam entity.TaskParam) error {
+	panic("unimplemented")
```
Comment (adiatma85) [inline src/business/usecase/task/task.go:71]: Sing iki penak e di merge disek opo yok po?
Isok tak modify, cuma branch e gk ndek repo iki, tapi ndek repomu

File: env/docker-compose.yml (hunk 1 above)
Comment (wirazeta) [reply env/docker-compose.yml:31]: ok

File: src/business/usecase/task/task.go (hunk 2 above)
Comment (wirazeta) [reply src/business/usecase/task/task.go:71]: Itu jgn di merge dlu klo bisa, masih mau ku edit

Comment (adiatma85) [reply src/business/usecase/task/task.go:71]: Oalah oke Wir, lek misal wes fix, tag en ae

**Comment by adiatma85** [comment]
Take over

**Pull request closed without merging**